### ISSUE TITLE
rfc6265bis: ABNF should just define field *values* (not the whole field line)

### ISSUE THREAD

**Issue by reschke** [httpwg/http-extensions#1302]

**Comment by miketaylr** [comment]
@reschke can you provide some more context here? I don't follow. Thanks.

**Comment by reschke** [comment]
From <https://greenbytes.de/tech/webdav/draft-ietf-httpbis-rfc6265bis-07.html#abnf-syntax>:

```
set-cookie-header = "Set-Cookie:" SP BWS set-cookie-string
set-cookie-string = BWS cookie-pair *( BWS ";" OWS cookie-av )
```

This defines ABNF for a field *line*. This is bad, because field line parsing is generic, and furthermore, there are no field lines in HTTP/2 or HTTP/3.

Just define the ABNF for the field value, consistent with the specs we wrote since ~2014.

**Comment by chlily1** [comment]
Closing, addressed by #1469
